packages/protocol/contracts/common/Proxy.sol

```diff
@@ -32,8 +33,16 @@ contract Proxy {
   function () external payable {
     bytes32 implementationPosition = IMPLEMENTATION_POSITION;
 
+    address implementationAddress;
+
+    assembly {
+      implementationAddress := sload(implementationPosition)
+    }
+
+    require(
```

Comment:
Style: if line needs to be split (over 100 characters), each argument on new line, put `);` on new line.